tests/ConfigurationTest.php

```diff
@@ -23,17 +23,6 @@ public function testDoesNotAcceptBadApiKey()
         new Configuration([]);
     }
 
-    public function testDefaultEndpoint()
-    {
-        $this->assertSame($this->config->getNotifyEndpoint(), 'https://notify.bugsnag.com');
```

Comment:
oh, I mean testing that our default instance base uri == `https://notify.bugsnag.com`. A typo in our constant or something should be caught.